README.md

```diff
@@ -23,22 +23,22 @@ Skopeo works with API V2 container image registries such as [docker.io](https://
 
  Skopeo operates on the following image and repository types:
 
- * containers-storage:docker-reference
+ * <i>containers-storage:docker-reference</i><br>
```

Comment:
I like the end result, but I'd rather not add html.  Could you change this to the md equivalents please?

```suggestion
 * **containers-storage:docker-reference**  
```
two astericks on either side of the word, then two white spaces at the end of the line to force `<br>`

Reply:
and ditto below.